pom.xml

```diff
@@ -106,15 +106,15 @@ flexible messaging model and an intuitive client API.</description>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <bookkeeper.version>4.3.1.82-yahoo</bookkeeper.version>
+    <bookkeeper.version>4.3.1.83-yahoo</bookkeeper.version>
```

Comment:
I didn't realize that we incremented bk version from `4.3.1.72-yahoo` to `4.3.1.82-yahoo`  in #689. In past it was always useful to know BK-changes in the PR when we have incremented bk-version to confirm changes in case of any issue (as it's tricky in yahooBK to know changes for a specific version).
is it fine to mention changes or just commit-range from [f87e0eda574](https://github.com/yahoo/bookkeeper/commit/f87e0eda574fb5f9d22dd7d92422eb221199a92c) to [165971209](https://github.com/yahoo/bookkeeper/commit/165971209f933e0bcefaac997db8e75c8776f942) in PR description.

Reply:
Yes, the increment there was needed because the berry version needs to be the same across BK and Pulsar at this point. 

Since 4.3.1.83 was already with netty 4.1, I had to first get broker to work with 4.1. 

The BK version change should be mentioned in one of the commits and should be there in the log of the squashed commit that was merged.

Reply:
Oh, I see. Yeah we should probably mention the changes in the bk version